website/docs/cloud-docs/architectural-details/security-model.mdx

```diff
@@ -1,46 +1,46 @@
 ---
-page_title: Security Model - Terraform Cloud
+page_title: Security model - Terraform Cloud
```

Comment:
Sorry, I should have clarified that page metadata should stay title case:
```suggestion
page_title: Security Model - Terraform Cloud
```

Reply:
Updated.  The style guide is a little confusing if that's that case.
> Use sentence case for page titles and all section headings on a page.